src/fixup.js

```diff
@@ -193,9 +192,5 @@
     document.body.appendChild(node);
   };
 
-  request.onerror = function() {
```

Comment:
this seems a little odd... `onerror` would only fire if the is a network error (i.e., the user disconnected from the network), not a 404?

Reply:
it definitely was triggering the error on 404 - I haven't looked into whether that was the expected behavior or not

Reply:
did you get the error while trying to generate the snapshot or just by opening the respec source document?

Reply:
while generating, in the respec source, and also in the a TR document loading fixup.js

Reply:
I can't seem to reproduce. Using the following code while at, for example, google.com:

```JS
var xhr = new XMLHttpRequest(); 
xhr.open("GET", "/asdfadasdffads");  
xhr.onerror = (err) => {console.log("err", err)};  
xhr.onload = ()=> {console.log("🎉", xhr.status)}; 
xhr.send(); 
```

I always get back "🎉 404".  The `onerror` never triggers. 

![screenshot 2017-10-03 19 46 11](https://user-images.githubusercontent.com/870154/31116923-8f45b3ea-a873-11e7-97e7-1531edc88406.png)

Reply:
I'm wondering if the error didn't happen while I was deploying the `tr-outdate-spec` service. I didn't think about it at first because @dontcallmedom mentioned the problem ~30min after the deployment but as a matter of fact, I can't reproduce the error locally.

Reply:
@dontcallmedom, maybe you hit:

```
XMLHttpRequest cannot load https://www.w3.org/TR/tr-outdated-spec. No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'null' is therefore not allowed access. The response had HTTP status code 404.
```

Reply:
That's expected, if loading something locally or from github, for instance.

Reply:
clearly we should avoid this CORS error when loading editors draft, right? But AFAIR, I had also hit an error in /TR/webrtc/ where this shouldn't have been an issue - maybe this was a temporary deployment issue, not sure how to tell.

Reply:
Although that seems 2 separate issues, how about I add the `request.onerror` back and call `/TR/tr-outdated-spec` only if the spec is loaded on w3.org?

Reply:
sounds worth trying :)